tests/unit/via/views/view_video_test.py

```diff
@@ -57,21 +60,23 @@ def test_it_with_YouTube_transcripts_disabled(
         with pytest.raises(HTTPUnauthorized):
             youtube(pyramid_request)
 
-    @pytest.fixture
-    def Configuration(self, patch):
-        Configuration = patch("via.views.view_video.Configuration")
-        Configuration.extract_from_params.return_value = (
-            sentinel.via_config,
-            sentinel.h_config,
-        )
-
-        return Configuration
-
     @pytest.fixture
     def video_url(self):
         return "https://example.com/watch?v=VIDEO_ID"
 
     @pytest.fixture
     def pyramid_request(self, pyramid_request, video_url):
-        pyramid_request.params["url"] = video_url
+        pyramid_request.params.update(
+            {"url": video_url, "via.foo": "foo", "via.bar": "bar"}
+        )
         return pyramid_request
+
+
+@pytest.fixture(autouse=True)
+def Configuration(patch):
+    Configuration = patch("via.views.view_video.Configuration")
+    Configuration.extract_from_params.return_value = (
+        sentinel.via_config,
+        sentinel.h_config,
+    )
+    return Configuration
```

Comment:
Moved this fixture to the bottom of the file (to make it global) and added `autouse=True` so that all tests in this file definitely use the mock `Configuration` object. Otherwise you can end up with tests that should fail passing because they're accidentally calling the real h-vialib